src/order/well_founded.lean

```diff
@@ -20,6 +20,25 @@ and provide a few new definitions: `well_founded.min`, `well_founded.sup`, and `
 variables {α : Type*}
 
 namespace well_founded
+
+theorem not_gt_of_lt {r : α → α → Prop} (h : well_founded r) :
```

Comment:
Let's add the `{α}` back here and below, I hadn't noticed it matched the rest of the file.

Reply:
Surely that just indicates it should be removed from the rest of the file? Would make a quick PR.

Reply:
#13702

Reply:
This lemma still needs changing, as it's true more generally for `α' : Sort`

Reply:
Would be nice to have a `Type` to `Sort` linter, on that note.